docs/cocos2d-x/sdk-configuration-android.mdx

```diff
@@ -702,7 +680,6 @@ Example:
 <TabItem value="For Cocos2d-x 3.x " label="For Cocos2d-x 3.x ">
 
 ```cppp
```

Comment:
This should be `cpp`
Please check if there are other such instances where the code block value is incorrect.